README.md

```diff
@@ -1,5 +1,68 @@
 # curriculum-cr-training-oop
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+This repo host code for the tasks performed during the Code Reviewers training.
 
-Application is based on the Ruby OOP: school library projects 1 and 2.
+Application is based on the Ruby OOP: school library projects 1, 2 and 3.
+
+## Getting started
+
+- To run the project you must have Ruby install on your computer. You can downloand and install Ruby from the link below.
+- windows [Ruby installer](https://rubyinstaller.org/).
+- MAC and LINUX [Ruby official site](https://www.ruby-lang.org/en/downloads/).
+
+## Once Ruby is download and installed, please follow the instructions bellow
+
+### 1- Clone the repo in your local folder where you want to run it
+
+- Run the command below in your terminal
+`git clone https://github.com/icebox827/oop-CR-training`.
+
+### 2- Install bundle
+
+- Run `bundle install` to install the gems from the `Gemfile`.
+
+### 3- Run the project
+
+- cd into `oop-CR-training/example_school_library_decorator` directory and checkout `feature/addBasicUI` branch then,
+- Run `ruby main.rb` to start the bot.
+- Then chose from the list of options displayed.
+
+## Built With
+
+- Ruby
+- Rubocop
+- Rspec
+- VsCode
+
+## Authors
+
+👤 **Denis Lafontant**
+
+- GitHub: [@icebox827](https://github.com/icebox827)
+- Twitter: [@heracles2k5](https://twitter.com/@heracles2k5)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/denis-lafontant-37031439/)
+
+👤 **Mih Julius**
+
+- Github: [@Mihndim2020](https://github.com/Mihndim2020)
+- Twitter: [@mihndim](https://github.com/mih-julius)
+- Linkedin: [Mih Julius](https://www.linkedin.com/mih-julius)
+
+
```

Comment:
- [ ] **REQUIRED:** It will be nice to add your personal information in the README file